src/card/aggregate-status/aggregate-status-card.component.js

```diff
@@ -30,6 +30,9 @@
  *   </ul>
  * </ul>
  * @param {boolean=} show-top-border Show/hide the top border, true shows top border, false (default) hides top border
+ * @param {boolean=} showSpinner Show/Hide the spinner for loading state. True shows the spinner, false (default) hides the spinner
+ * @param {string=} spinnerText Text for the card spinner
+ * @param {string=} loadingCardHeight Height to set for the card when data is loading
```

Comment:
We use loading here but showSpinner. I think it needs to be named such that it is obvious that this is the card height when showing the spinner.

Reply:
good point.   spinnerCardHeight?

Reply:
I think renaming to spinnerCardHeight is good to show the correlation